sos/report/plugins/ebpf.py

```diff
@@ -22,27 +22,29 @@ class Ebpf(Plugin, IndependentPlugin):
     ]
 
     def get_bpftool_prog_ids(self, prog_json):
+        """ Collect the list of program IDs """
         out = []
         try:
             prog_data = json.loads(prog_json)
-        except Exception as e:
-            self._log_info("Could not parse bpftool prog list as JSON: %s" % e)
+        except Exception as err:  # pylint: disable=broad-except
```

Comment:
Address [C0200](https://pylint.readthedocs.io/en/stable/user_guide/messages/convention/consider-using-enumerate.html).

I've verified that the changes are functionally equivalent.